packages/eas-cli/src/commands/build/index.ts

```diff
@@ -95,6 +95,10 @@ export default class Build extends Command {
 
       await ensureProjectConfiguredAsync(projectDir);
```

Comment:
```suggestion
      exp = await ensureProjectConfiguredAsync(projectDir, exp);
```

ensureProjectConfiguredAsync runs configuration only in one very specific case, but this solution is reading config twice every time, maybe move it to internals of that functions?